powerquery-docs/Connectors/DatabricksCloud.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Databricks connector
+description: Provides basic information and prerequisites for using Power Query's Databricks connectors.
+author: George Chow
+
+ms.topic: conceptual
+ms.date: 6/29/2022
+ms.author: George Chow
+LocalizationGroup: reference
+---
+
+# Databricks Cloud
+
+## Summary
+
+| Item | Description |
+| ---- | ----------- |
+| Release State | Azure Databricks: General Availability<br/>Databricks: Preview |
+| Products | Power BI (Datasets)<br/>Power BI (Dataflows) |
+| Authentication Types Supported | Azure Active Directory<br/>Organizational Account<br/>Personal Access Token |
+| | |
+
+>[!Note]
+>Some capabilities may be present in one product but not others due to deployment schedules and host-specific capabilities.
+
+
+## Prerequisites
+
+You requires either an Azure Databricks account or a paid Databricks Cloud account to use this connector. Databricks Community Edition is not supported.
+
+## Capabilities supported
+
+* Import
+* DirectQuery (PowerBI only)
+
+## Connect to Azure Databricks or Databricks Cloud from Power Query Desktop
+
+To connect from Power BI Desktop, take the following steps:
+
+1. In the Get Data experience, search for **databricks** to shortlist the two Databrick connectors: "Azure Databricks" and "Databricks". Choose "Azure Databricks" if your Databricks runs in the Azure cloud; for AWS or GCP, choose "Databricks". (If the entry for "Databricks" is marked with a "(Beta)" suffix, contact your Databricks representative for guidance on using the connector.)
```

Comment:
Azure Databricks connector can be used for connecting to Databricks on AWS. and many customers do that today.

Azure Databricks connector is GA but the other connector is not GA yet (it is beta) hence in the public documentation we should promote Azure Databricks connector for all clouds. We should just mention at the end that there is a beta generic connector with private preview support for OAuth on AWS. However till beta tag is not removed we shouldn't promote this connector for use in prod yet.

Reply:
k